### DISCUSSION TITLE
App crashes on notification arrival in iOS

### DISCUSSION THREAD

**Discussion by Yahya0007** [shinyorg/shiny#1474] [Q&A]
Hi

This works fine on Android. The issue is on iOS.

When notification arrives OnReceived in push delegate is called fine. However, before OnEntry is called the app crashes.

Sentry screen for the crash is attached. 

I am having trouble getting to the root cause and fix for this issue. Would any help.

Thanks 

Regards

![Shinny-Sentry-3](https://github.com/shinyorg/shiny/assets/50534591/f8e3a9df-80eb-4333-9fe4-edd62e7f1d79)

**Comment by aritchie** [comment]
This is not where issues go.  You need to provide a sample and all of the details required in the issue template
